src/js/profile/mobile/widget/Calendar.js

```diff
@@ -0,0 +1,275 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable block-scoped-var */
+/* eslint-disable no-eq-null */
+/* eslint-disable one-var */
+/* eslint-disable vars-on-top */
+/*global window, define, ns */
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * @example
+ * <div class="ui-calendar"></div>
+ *
+ * @since 1.2
+ * @class ns.widget.core.Calendar
+ * @extends ns.widget.BaseWidget
+ * @author Dohyung Lim <delight.lim@samsung.com>
+ */
+(function (document, ns) {
+	"use strict";
+	//>>excludeStart("tauBuildExclude", pragmas.tauBuildExclude);
+	define(
+		[
+			"../../../core/engine",
+			"../../../core/util/DOM/attributes",
+			"../widget"
+		],
+		function () {
+			//>>excludeEnd("tauBuildExclude");
+			var utilsObject = ns.util.object,
+				engine = ns.engine,
+				events = ns.event,
+				weeks = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"],
+				fullNameMonth = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
+				today = new Date(),
+				todayYear = today.getFullYear(),
+				todayMonth = today.getMonth() + 1,
+
+				Calendar = function () {
+					this.options = utilsObject.merge({}, Calendar.defaults);
+				},
+
+				defaults = {
+					value: false // false is past selectable, true is past selection not possible.
+				},
+
+				BaseWidget = ns.widget.BaseWidget,
+				prototype = new BaseWidget();
+
+			Calendar.prototype = prototype;
+			Calendar.defaults = defaults;
+
+			prototype._init = function (element) {
```

Comment:
How about adding the description for the api?

Reply:
Add description to the required function.

done.